tests/pflang-reg/pl-bug132-icmp6_or_portrange

```diff
@@ -0,0 +1 @@
+../../env pipe-lua-libpcap-match ../data/wingolog.pcap "icmp6 or portrange 14682-50101" 14951
```

Comment:
I think this one and the next needs the shebang (#!/bin/bash).
If it's the same to you, let's include the "env" trampoline in the makefile rather than the test.  That way you can run the tests even if you don't happen to be in the pflang-reg directory.